fall18/p2/README.md

```diff
@@ -1,3 +1,149 @@
+  Problem 12: function predictPopulation not found
+  Problem 13: function predictPopulation not found
+  Problem 14: function predictPopulation not found
+  Problem 15: function predictPopulation not found
+  Problem 16: function calcGrowthRate not found
+  Problem 17: function calcGrowthRate not found
+  Problem 18: function calcGrowthRate not found
+  Problem 19: function calcGrowthRate not found
+  Problem 20: function calcGrowthRate not found
+```
+
+##  Introduction
+We provide you with a data file containing information about 50 states in the United States. Our task for this assignment is to look at the table and try to answer the questions that follow. We are going to write a few functions in order to make these calculations.
+
+The table below shows a **sample** from the dataset that we have. 
+
+<center>
+
+| State | Area (mi²) | Population (2000) | Population (2010) | population (2014) |
+|------|------|------|------|------|
+|Wisconsin|65496.38 |5363675|5686986|5759432|
+|Minnesota|86935.83 |44919479|5303925|5457125|
+|Illinois|57913.55 |12419293|12830632|12882189|
+
+</center>
+
+1. What is the value of the maximum land area among any three given states?
+2. What is the least value of population density among any three given states?
+3. For a given state, what’s the predicted population in the given year? (The function will take in values of initial year, final year, growth rate and state name)
+4. Between any two given years, what is the population growth rate of particular state?
+
+### Provided Function in Module project.py
+We don’t require you to know how to read from data file so we have provided the following functions. Please make sure the data file area.csv and population.csv are in the same folder along with main.py.
+
+**getArea(stateName)** 
+
+getArea function takes the name of one state as an argument and returns the size of this state area.
+
+```
+>>> import project
+>>> project.getArea("Wisconsin")
+>>> 65496.38
+```
+**getPopulation(stateName, year)**
+
+getPopulation function takes the name of one state and a year as arguments and returns the population of this state in this year.
+
+```
+>>> import project
+>>> project.getPopulation("Wisconsin", 2000)
+>>> 5363675
+```
+
+## Program Requirements
+You will write **at least four(4)** functions with the following names and behaviors:
+
+**getMaximumLand(stateName1, stateName2, stateName3)**
+
+Find the max value for land area among the three given area values which will be passed as parameters. To do this we can use the inbuilt [max()](https://docs.python.org/2/library/functions.html#max) function.
+
+Example:
+
+```
+>>> getMaximumLand("Wisconsin", "Iowa", "Minnesota")
+>>> 86935.83
+```
+
+**getMinimumPopulationDensity(stateName1, stateName2, stateName3, year)**
+
+Population density is measured as population per unit area. 
+
+The formula to calculate population density is:
+
+**Population Density = Population / Land Area (mi²)**
```

Comment:
It's weird that this has the same styling as the four functions.  Maybe only the four functions should be bold?  Or, separate them with some extra text, like this:

Function 1: getMaximumLand(...)

Reply:
I agree. Please remove the bold from the formula for Population Density and add a prefix (e.g., Function 1) before the functions.